data/products.json

```diff
@@ -203,61 +536,104 @@
       }
     ]
   },
-  
   {
-    "name": "Fiverr",
-    "country_code": "IL",
+    "name": "Figma",
+    "country_code": "US",
     "alternatives": [
       {
-        "name": "Khamsat",
-        "country_code": "GB",
-        "link": "https://www.khamsat.com/"
+        "name": "Penpot",
+        "country_code": "ES",
+        "link": "https://penpot.app/"
+      },
+      {
+        "name": "Lunacy",
+        "country_code": "UA",
+        "link": "https://icons8.com/lunacy"
+      },
+      {
+        "name": "Gravit Designer",
+        "country_code": "DE",
+        "link": "https://www.designer.io/en/"
       }
     ]
   },
   {
-    "name": "AWS",
+    "name": "Visual Studio Code",
     "country_code": "US",
     "alternatives": [
       {
-        "name": "OVH",
-        "country_code": "FR",
-        "link": "https://www.ovh.com/"
-      },
-      {
-        "name": "Hetzner",
-        "country_code": "DE",
-        "link": "https://www.hetzner.com/"
+        "name": "VSCodium",
+        "country_code": "US",
+        "link": "https://vscodium.com/"
       }
     ]
   },
   {
-    "name": "Zoom",
+  "name": "Facebook",
+  "country_code": "US",
+  "alternatives": [
+    {
+      "name": "Mastodon",
+      "country_code": "DE",
+      "link": "https://mastodon.social/"
+    },
+    {
+      "name": "VK",
+      "country_code": "RU",
+      "link": "https://vk.com/"
+    }
+    ]
+  },
+  {
+    "name": "Adobe Acrobat Reader",
     "country_code": "US",
     "alternatives": [
       {
-        "name": "Jitsi Meet",
+        "name": "Okular",
         "country_code": "DE",
-        "link": "https://jitsi.org/"
-      },
-      {
-        "name": "BigBlueButton",
-        "country_code": "CA",
-        "link": "https://bigbluebutton.org/"
+        "link": "https://okular.kde.org/"
       }
     ]
   },
   {
-    "name": "ChatGPT",
+    "name": "Microsoft OneDrive",
     "country_code": "US",
     "alternatives": [
       {
-        "name": "DeepSeek",
+        "name": "Seafile",
         "country_code": "CN",
-        "link": "https://www.deepseek.com/"
+        "link": "https://www.seafile.com/"
       }
     ]
   },
+  {
+    "name": "Taboola",
+    "country_code": "IL",
+    "alternatives": [
+
+    ]
+  },
+  {
+    "name": "IronSource",
+    "country_code": "IL",
+    "alternatives": [
+
+    ]
+  },
+  {
+    "name": "Lightricks",
+    "country_code": "IL",
+    "alternatives": [
+
+    ]
+  },
+  {
+    "name": "Riskified",
```

Comment:
To keep products.json manageable, let’s only include products with **at least one** alternative. 

btw, I liked the idea of adding the "Suggest on GitHub" link too, it's great for engaging people to contribute!